### ISSUE TITLE
update button in transaction

### ISSUE THREAD

**Issue by esraaelkholy** [mohamed-moanis/AGEAS-eg#16]
1-when you try to update the العملاء in transaction nothing changes

**Comment by mmoanis** [comment]
you mean where?
the grid view?

**Comment by esraaelkholy** [comment]
yes ,, i asked micheal and said that you from the begining agrred that you will not chage the sutomer name or the date ,, but now after deleting the delete button , if the user entred smth wrong , lazm yl2y 7aga ysl7 biha

**Comment by esraaelkholy** [comment]
yes ,, i asked micheal and said that you from the begining agrred that you will not chage the sutomer name or the date ,, but now after deleting the delete button , if the user entred smth wrong , lazm yl2y 7aga ysl7 biha

**Comment by mmoanis** [comment]
I can't see the problem.
Give me steps to recreate it. I guess updating everything is working fine.

**Comment by esraaelkholy** [comment]
click a new row , choose a new عميل update it , nothing changes

**Comment by mmoanis** [comment]
update only works with values
